client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx

```diff
@@ -13,23 +13,27 @@ import {ExposureAndFlightsContextProvider, ExposureAndFlightsDetails,
 
 import useStyles from './InvestigationFormStyles';
 import useInvestigationForm from './useInvestigationForm';
+import TabManagement from './TabManagement/TabManagement';
+import ConvesrationScript from './ConversationScript/ConvesrationScript';
+import InvestigationInfoBar from './InvestigationInfo/InvestigationInfoBar';
 import useGroupedInvestigationContacts from './useGroupedInvestigationContacts';
 import useTabManagement ,{ LAST_TAB_ID } from './TabManagement/useTabManagement';
-import InvestigationInfoBar from './InvestigationInfo/InvestigationInfoBar';
-import TabManagement from './TabManagement/TabManagement';
 import TrackingRecommendationForm from './TrackingRecommendation/TrackingRecommendationForm';
 import { StartInvestigationDateVariablesProvider } from './StartInvestiationDateVariables/StartInvestigationDateVariables';
 
 const InvestigationForm: React.FC = (): JSX.Element => {
     const classes = useStyles({});
 
+    const initialSctiptState = ( localStorage.getItem('isScriptOpened') ?? 'true' ) === 'true';
```

Comment:
Please notice that localStorage can only accept strings (hence 'true' and 'false')